src/docs/app-monitoring/user-defined-monitoring.md

```diff
@@ -212,9 +212,43 @@ For detail steps, please read the documents below:
 
 ## Monitor user-defined projects using Alertmanager
 
-Technical preview for our Openshift Clusters (OCP 4.10). It is possible to set up Alertmanager rule for a user-defined projects so that the granted user(s) by the monitoring-rules-edit role can create, modify, and deleting PrometheusRule custom resources for their project. It's coming with OCP 4.11.
+Alerting rules are created for user-defined projects and based on specific chosen metrics these alerts get triggered.
```

Comment:
I don't think people reading this will be thinking about the difference between infra and user projects. We also try and keep the `Project` keyword out of things as it has a special meaning in BCGov. `namespace` is better.

Reply:
Can we add more language to tie this to the previous section that created the custom metric? Can we also explain that you can use the non-custom metrics that the platform already has for your alerts?

Reply:
Can we add some info about the `AlertmanagerConfig` object that gets created for teams in their namespaces? Maybe also link to https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/docs/platform-automation/alertmanager/

Reply:
converted project to namespace in doc and added language to tie up previous section.